examples/connext_dds/asynchronous_publication/cs/README.md

```diff
@@ -1,5 +1,9 @@
 # Example Code: Asynchronous Publication
 
+:warning: **Note**: This example uses the **Legacy .NET API**. Until it is
+updated, you can learn more about the **new Connext DDS C# API** in the
+[Getting Started Guide](https://community.rti.com/static/documentation/connext-dds/current/doc/manuals/connext_dds/getting_started/index.html).
```

Comment:
You could use https://community.rti.com/static/documentation/connext-dds/6.1.0/doc/manuals/connext_dds_professional/getting_started_guide/csharp/before_csharp.html instead.

We can always do a bulk change when we bump the version, but the link to current is going to break.